utils/webassembly/build-toolchain.sh

```diff
@@ -0,0 +1,61 @@
+#/bin/bash
+
+set -x
+SOURCE_PATH="$(cd "$(dirname $0)/../../.." && pwd)"
+UTILS_PATH="$(cd "$(dirname $0)" && pwd)"
+
+WASI_SDK_PATH=$SOURCE_PATH/wasi-sdk
+
+case $(uname -s) in
+  Darwin)
+    OS_SUFFIX=osx
+    BUILD_SCRIPT=$UTILS_PATH/build-mac.sh
+  ;;
+  Linux)
+    OS_SUFFIX=linux
+    BUILD_SCRIPT=$UTILS_PATH/build-linux.sh
+  ;;
+  *)
+    echo "Unrecognised platform $(uname -s)"
+    exit 1
+  ;;
+esac
+
+YEAR=$(date +"%Y")
+MONTH=$(date +"%m")
+DAY=$(date +"%d")
+TOOLCHAIN_VERSION="5.2.${YEAR}${MONTH}${DAY}"
```

Comment:
I'm thinking it's 5.3 at this point, at least Swift Evolution refers to recently implemented proposals as "implemented in 5.3".

Reply:
I omitted version prefix following swift-tensorflow manner.